### ISSUE TITLE
LongTypeHintsSniff fails to fix Boolean

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#366]
The following code fails causes the sniff to error out:
```php
/**
 * @param Boolean $foo description
 */
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/c93f5bb6dbc15f6b1adcbba47d7c276e151ec50d
